README.md

```diff
@@ -284,6 +300,169 @@ el "button" [attr "type" "button", onClick_ save] do
   text "Save"
 ```
 
+For inputs we have predefined props that make `change` and `input` events handling easier (available in [Specular.Dom.Element](https://pursuit.purescript.org/packages/purescript-specular/docs/Specular.Dom.Element))
+
+```purescript
+-- * Input value
+-- | Attach dynamically-changing `value` property to an input element.
+-- | The value can still be changed by user interaction.
+-- |
+-- | Only works on `<input>` and `<select>` elements.
+valueD :: Dynamic String -> Prop
+
+-- | Set up a two-way binding between the `value` of an `<input>` element,
+-- | and the given `Ref`.
+-- |
+-- | The `Ref` will be updated on `change` event, i.e. at the end of user inteaction, not on every keystroke.
+-- |
+-- | Only works on input elements.
+bindValueOnChange :: Ref String -> Prop
+
+
+-- | Attach dynamically-changing `checked` property to an input element.
+-- | The value can still be changed by user interaction.
+-- |
+-- | Only works on input `type="checkbox"` and `type="radio"` elements.
+checkedD :: Dynamic Boolean -> Prop
+
+-- | Set up a two-way binding between the `checked` of an `<input>` element,
+-- | and the given `Ref`.
+-- |
+-- | Only works on input `type="checkbox"` and `type="radio"` elements.
+bindChecked :: Ref Boolean -> Prop
+
+```
+
+Example: 
+
+```purescript
+
+import Prelude
+import Specular.Ref (Ref, newRef)
+import Specular.Dom.Browser ((:=))
+import Specular.Dom.Element (el, attrs, bindValueOnChange)
+import Specular.Dom.Widget (emptyWidget)
+
+let description :: Ref String = newRef ""
+
+el "input" [attrs ("type" := "text")    , bindValueOnChange description] emptyWidget
+
+```
+
+
+#### A Counter example
+
+```purescript
+module Main where
+
+import Prelude
+import Effect (Effect)
+
+import Data.Functor.Contravariant (cmap)
+
+import Specular.Callback (mkCallback, triggerCallback)
+import Specular.Dom.Browser ((:=))
+import Specular.Dom.Element (attrs, class_,  el,  onClick_, text, dynText)
+import Specular.Dom.Widget (runMainWidgetInBody)
+import Specular.Ref (Ref, value, newRef, modify)
+
+
+
+main :: Effect Unit
+main = do
+  -- | Will append widget to the body
+  runMainWidgetInBody do
+    counter :: Ref Int <- newRef 0
+  
+    -- | Subtract 1 from counter value the straight forward way
+    let subtractCb = mkCallback \_ -> triggerCallback (modify counter) (add (negate 1))
+
+    -- | Add 1 to counter value using the contravariant instance
+    let addCb = cmap (\_ -> add 1) (modify counter)
+
+    el "button" [class_ "btn", attrs ("type":="button" ), onClick_ addCb ] do
+      text "+"
+
+    dynText $ show <$> value counter
+
+    el "button" [class_ "btn", attrs ("type":="button" ), onClick_ subtractCb ] do
+      text "-"
+```
+
+<p class="callout warning">Warning: examples which can be found in this repo which are using "FixFRP" are deprecated !</p>
+
+
+## Getting started using spago
```

Comment:
Maybe make a repo out of this?

Reply:
Done https://github.com/restaumatic/purescript-specular-starter, also added to readme.